test/test_main.py

```diff
@@ -787,16 +781,19 @@ def test_hook_and_dispatch_with_failing_hook(self):
         self.stdout.seek(0)
         self.assertEqual(self.stdout.read(), b'')
         calls = fake_script_calls(self)
+        hook = Path('hooks/install')
         expected = [
             VERSION_LOGLINE,
-            ['juju-log', '--log-level', 'INFO', 'Running legacy hooks/install.'],
-            ['juju-log', '--log-level', 'WARNING', 'Legacy hooks/install exited with status 42.'],
+            ['juju-log', '--log-level', 'INFO', 'Running legacy %s.' % (hook,)],
+            ['juju-log', '--log-level', 'WARNING', 'Legacy %s exited with status 42.' % (hook,)],
```

Comment:
Why not `.format()`?

Reply:
more typing? :-p
(probably just muscle memory)